lms/envs/common.py

```diff
@@ -2268,6 +2268,7 @@
 STATIC_TEMPLATE_VIEW_DEFAULT_FILE_EXTENSION = 'html'
 
 SUPPORT_SITE_LINK = ''
+ACTIVATION_EMAIL_SUPPORT_LINK = 'https://support.edx.org/hc/en-us/articles/227340127-Why-haven-t-I-received-my-activation-email-'
```

Comment:
Ok, so this will default to `''` for now, while defaulting to `SUPPORT_SITE_LINK` elsewhere if it isn't set? Because presumably, most installs are currently only familiar with `SUPPORT_SITE_LINK`.